src/main/java/org/micks/champmaker/championships/ChampionshipService.java

```diff
@@ -51,6 +51,19 @@ public void createChampionship(ChampionshipDTO championshipDTO) {
         championshipRepository.save(championshipEntity);
     }
 
+    public void editingChampionship(long champId, ChampionshipDTO championshipDTO) throws EntityNotFoundException {
+        Optional<ChampionshipEntity> editChamps = championshipRepository.findById(champId);
```

Comment:
Variable name `editChamps` is confusing. Maybe something like `optionalChampionship`?

Reply:
corrected